lib/iris/experimental/representation.py

```diff
@@ -102,13 +102,22 @@ def __init__(self, cube):
             'Dimension coordinates:': None,
             'Auxiliary coordinates:': None,
             'Derived coordinates:': None,
+            'Cell Measures:': None,
+            'Ancillary Datasets:': None,
```

Comment:
We have been throwing around a few different names for this (Ancillary Data, Ancillary Dataset, Ancillary Variable), so I think that's where some of the confusion has come from, but have agreed to call these `Ancillary Variables` so this will need updating

Reply:
Ancillary Datasets is what is currently being written in the summary here https://github.com/SciTools/iris/blob/77e4b3f7df83eb20a56957e236b7dc9073ce5458/lib/iris/cube.py#L2206 . Should I take it that this ought to be changed?

Reply:
Good spot! That does need changing!
I had originally called in "Ancillary Datasets" but that was pushed back so I needed to rename everything and clearly missed that name change!